helm/charts/hydra/templates/ingress-admin.yaml

```diff
@@ -1,41 +1,52 @@
 {{- if .Values.ingress.admin.enabled -}}
 {{- $fullName := include "hydra.fullname" . -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: networking.k8s.io/v1
 {{- else -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
 {{- end }}
 kind: Ingress
 metadata:
   name: {{ $fullName }}-admin
   namespace: {{ .Release.Namespace }}
   labels:
-{{ include "hydra.labels" . | indent 4 }}
+    {{- include "hydra.labels" . | nindent 4 }}
   {{- with .Values.ingress.admin.annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-{{- if .Values.ingress.admin.tls }}
+  ingressClassName: {{ .Values.ingress.admin.className }}
```

Comment:
Currently we have 
```yaml
annotations: {}
    #     kubernetes.io/ingress.class: nginx
```
Maybe we could use this here? Add a function checking the annotations and set the field as 
```yaml
ingressClassName: {{ include "hydra.extractClassFromAnnotations" .  | default  .Values.ingress.admin.className }}
```

Reply:
I think that since those who want to update to the new version of the helm chart must change, in any case, the values of the ingress (`paths` array) it is not convenient to make this function. WYT?

Reply:
True enough, in that case, could you write as a comment what will be required during the upgrade process, from the user? Something we could add to the release notes/readme